.github/workflows/label-analysis.yml

```diff
@@ -32,6 +38,8 @@ jobs:
           done
           echo "PR ${{github.event.number}} requires at least one non-triage team assignment label (label starting by 'team/')"
           exit 1
+        env:
+          LABELS: ${{ steps.pr-labels.outputs.MILESTONE }}
```

Comment:
Shouldn't this be `outputs.LABELS`?

Reply:
The way you access the labels looks weird to me. Shouldn't it be something like `jobs.fetch-labels.outputs.**`?

Reply:
@chouquette good point, forgot to fix my copy paste aha, fixed in [9718176](https://github.com/DataDog/datadog-agent/pull/22657/commits/97181767da326ad97df91a600953d2974e0bf4af) !

Reply:
@KevinFairise2 you're right, nice catch ! Fixed in [c5a1456](https://github.com/DataDog/datadog-agent/pull/22657/commits/c5a1456a88b75aea88e4d6f5406c132c9d577215).